datamade/just-spaces
# Draft data import pipeline

## Overview

Draft a simple data import pipeline for extracting a variety of ACS data for the app.

### Notes

For now, I didn't put the raw data under version control, since we're not totally sure what intermediate/final data will look like yet. I figure it would make sense to do that once the full pipeline is in a semi-finished state.

## Testing Instructions

* Follow the instructions in the README to make the data: https://github.com/datamade/just-spaces/blob/jfc/data-import-pipeline%2367/data/README.md
* Do a quick sniff of the data to make sure that it looks reasonable

Closes #67
